i18n/en.json

```diff
@@ -4424,7 +4424,7 @@
   "pricing_modal.briefing.professional.advancedPlaybook": "Advanced Playbook workflows with retrospectives",
   "pricing_modal.briefing.professional.messageBoardsIntegrationsCalls": "Unlimited access to messages and boards history, teams, and calls",
   "pricing_modal.briefing.ssoWithGitLab": "SSO with Gitlab",
-  "pricing_modal.briefing.storage": "{storage} file storage limit",
+  "pricing_modal.briefing.storage": "Unlimited storage limit",
```

Comment:
Actually, this key is used in both pro and starter limits. So, this will break starter now which passes the `storage` value as param.
I guess you can either create separate keys for both starter and pro. Or just pass `Unlimited` when using with Pro. key instead of any actual storage value.